### DISCUSSION TITLE
How do you pronounce the "k3s"?

### DISCUSSION THREAD

**Discussion by inductor** [k3s-io/k3s#2969] [Q&A] [answered]
I remember seeing k3s can be expressed as `k "ate" s` somewhere on ibuildthecloud/k3s before, which is the exactly same pronunciation as k8s(`k "eight" s`), but I couldn't find it on any pages now 👀 

Is it `kates`, or `kes` like leet language?

**Comment by clearcodecn** [comment]
k3s, 比k8s少5s

**Comment by inductor** [comment]
Guys, I'm asking how to pronounce the name. I know why it's written as k3s 😂

**Comment by warmchang** [comment]
😂 Sorry for misunderstanding your meaning, the pronunciation of "k3s" seems to require rancher official to answer.

**Comment by wbright** [comment]
maybe kbeds :)   kubernetes for embeded device

**Comment by richardlewisjones** [comment]
Kubes, surely?

**Comment by inductor** [comment]
It's like lottery haha

**Comment by buschtoens** [comment]
Someone on HN suggested Kuberne[_tres_](https://en.wiktionary.org/wiki/tres#Spanish). 😂

**Comment by glikson** [comment]
Maybe like "K3", as if it didn't have an "s"?

**Comment by nijikokun** [comment]
`Kes` or `keys`

**Comment by ibuildthecloud** [comment]
I feel like the community should decide.  I vote for [Boaty McBoatFace](https://en.wikipedia.org/wiki/Boaty_McBoatface).

**Comment by snow-jallen** [comment]
...wouldn't it just be k-threes?  Although kubernetres is funny. 👍 
k-threes fits with the whole "five less than eight" idea.

**Comment by jmrodri** [comment]
I likes "Kubes" since "K 3 (ube) S" matches what K8S (k UBERNETE s).

**Comment by inductor** [comment]
How do you vote?

**Comment by Cheau** [comment]
I vote for `Kubes`.
The reasons are:
1. `3-ube` matches the word-building of `8-ubernete`
2. Kubes is easy to pronounce and seems like an abbreviation of Kubernetes
3. it's literally meaningful to represent itself as a variant of Kubernetes

**Comment by tdensmore** [comment]
My vote is for : 
  "Kay" "Three" "Ess"

**Comment by 06kellyjac** [comment]
Since kubernetes is Greek why not stick with the theme
κυβερνήτης (10 Letters) -> kubernetes, means Governor, Helmsman, Captain
κύβος (5 letters) -> kubos, means Cube

k8s is often pronounced as kates or k 8 s
k3s could be pronounced keys but since I usually go with k 8 s I'll probably say k 3 s

**Comment by inductor** [comment]
Looks like Kubes was the most voted?

**Comment by inductor** [comment]
@ibuildthecloud

**Comment by MatrixManAtYrService** [comment]
Just wanted to speak out against "Kubes" to differentiate it from "Qubes".

At least while talking to myself I'm gonna call it tetrahedrons, because in an odd way...

Qubes : K3s
VMs : containers
cubes : tetrahedrons (there's a little less to them)

**Comment by Himura2la** [comment]
So strange that there's still no canonical pronunciation. At least the dev team should call it some way when talking about it...

P.S.: I like Kubes [kjuːbs],

**Comment by cjellick** [comment, marked as answer]
I've only ever heard @ibuildthecloud (the creator) pronounce it as "kay three ess" :-D

**Reply by nunix** [reply]
I mean, kube see tee hell right?
So kay three ess is the right one

**Reply by clintmod** [reply]
creator should be the final say

**Comment by cjellick** [comment]
This is a lovely candidate for our first conversion of an issue to discussion 
😄
